diff -ru prepatch/subdir/bar postpatch/subdir/bar
--- prepatch/subdir/bar	2007-10-02 17:35:14.000000000 -0400
+++ postpatch/subdir/bar	2007-10-02 17:35:41.000000000 -0400
@@ -5,9 +5,9 @@
 5
 7
 8
-9
-10
-11
+a
+b
+c
 12
 13
 14
diff -ru prepatch/subdir/foo postpatch/subdir/foo
--- prepatch/subdir/foo	2007-10-02 17:34:58.000000000 -0400
+++ postpatch/subdir/foo	2007-10-02 17:35:47.000000000 -0400
@@ -5,7 +5,7 @@
 5
 6
 7
-8
+change
 9
 10
 11
