diff -r 8c59bbe1e7c5 conary/pysqlite3/Makefile
--- a/conary/pysqlite3/Makefile	Mon Sep 21 11:28:59 2009 -0400
+++ b/conary/pysqlite3/Makefile	Mon Sep 21 13:35:42 2009 -0400
@@ -6,6 +6,8 @@
 .PHONY: all dist clean install
 
 all:
+        # default (tested) sqlite to embed is 3.6.18
+	[ -f sqlite-3.6.18/.libs/libsqlite3.a ] && cp sqlite-3.6.18/.libs/libsqlite3.a . || :
 	$(PYTHON) setup.py build
 	$(PYTHON) setup.py install --install-lib=..
 
diff -r 8c59bbe1e7c5 conary/pysqlite3/setup.py
--- a/conary/pysqlite3/setup.py	Mon Sep 21 11:28:59 2009 -0400
+++ b/conary/pysqlite3/setup.py	Mon Sep 21 13:35:42 2009 -0400
@@ -11,7 +11,7 @@
 macros = []
 
 if sys.platform in ("linux-i386", "linux2"): # most Linux
-    include_dirs = []
+    include_dirs = ['sqlite-3.6.18/']
     library_dirs = []
     libraries = [sqlite]
     runtime_library_dirs = []
