diff -ur a/config/macros b/config/macros
--- a/config/macros	2007-06-20 17:01:54.000000000 -0400
+++ b/config/macros	2007-06-21 22:01:34.663901799 -0400
@@ -56,9 +56,11 @@
 cxxflags            = ''    # cxx specific flags
 optflags            = '-O2'
 dbgflags            = '-g' # for debuginfo
-cflags              = '%(optflags)s %(dbgflags)s' 
-cppflags	    = '' 
-ldflags		    = '%(dbgflags)s'
+securityflags       = '-D_FORTIFY_SOURCE=2 -fstack-protector'
+cflags              = '%(optflags)s %(dbgflags)s %(securityflags)s' 
+cppflags	    = ''
+ldoptflags          = '-O1 '
+ldflags             = '%(dbgflags)s %(ldoptflags)s'
 mflags		    = '' # make flags
 parallelmflags      = ''
 classpath           = ''
